codesy/templates/401.html

```diff
@@ -1 +1,13 @@
-<a href="https://{{ current_site.domain }}">Sign in to codesy.io</a>.
+{% load staticfiles %}
+<link rel="stylesheet" href="//{{current_site.domain}}{% static "css/styles.css" %}">
+<link rel="stylesheet" href="//cdn.jsdelivr.net/pure/0.5.0/pure-min.css">
```

Comment:
I'd rather file an issue.  I don't think this will be easy since the templates are in separate directories.  Ideally we'd put any extension related templates together.